css-layout-classes/styles.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
```

Comment:
For flexbox items, recommended to use `flex-basis` property instead of `width`. This will allow for more responsive layout as flexbox will also let you define how the item will grow/shrink as resizing is done...